### DISCUSSION TITLE
Conversation history cannot be retrieved

### DISCUSSION THREAD

**Discussion by tsuchiya-ryosuke** [langgenius/dify#11000] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

I am having trouble retrieving conversation history.
The API seems to be authenticated, but I wonder if the user_id is wrong.
If the user_id is wrong, can I check the user_id from the Didify screen?

```
import requests

url = 'https://api.dify.ai/v1/conversations'
params = {
    'user': 'bbd536bf-bbb4-4e48-9fc5-8814e4d2d4f6',
    # 'last_id': '',
    'limit': 20
}

api_key = 'my-api-key'
headers = {
    'Authorization': f'Bearer {api_key}'
}

response = requests.get(url, headers=headers, params=params)
print(response.json())
```

**Comment by yuguanggao** [comment]
Hi! I'd like to know what should be the 'api_key' for the Bearer authorization? Is it created at the api access session that looks something like "app-xxxxxxxxxrIvdFJoi7N5y"? But when I tried it, the output of response.json() is: {"code": "unauthorized", "message": "Access token is invalid", "status": 401}.
